packages/lib/modules/vebal/lock/form/VebalLockForm.tsx

```diff
@@ -70,7 +70,7 @@ export function VebalLockForm({ allowEditOnInit = false }: Props) {
     previewModalDisclosure.onClose()
     refetchAll()
     if (isSuccess) {
-      router.push('/vebal/manage')
+      router.push('/vebal/manage/')
```

Comment:
Does this on close always have to go to the manage page?

Reply:
I forgot to change this one. Done.